src/einsteinpy/metric/kerrnewman.py

```diff
@@ -0,0 +1,329 @@
+import warnings
+
+import astropy.units as u
+import numpy as np
+
+from einsteinpy import constant
+from einsteinpy.integrators import RK45
+from einsteinpy.utils import *
+from einsteinpy.utils import kerrnewman_utils, schwarzschild_radius
+
+_G = constant.G.value
+_c = constant.c.value
+_Cc = constant.coulombs_const.value
+# q = 11604461683.91822052001953125*q This comment is not to be removed. This is a hack.
```

Comment:
Hack for what?

Reply:
Charge needs a scaling factor to work, just like in schwarzschild. We have to multiply `11604461683.91822052001953125` with original `q` to get the correct dynamics as of now. I don't know where it came from. I tried to fix it, but no relief. So I figured, it would see the same fate as `Schwarzschild` module. I may figure it with time, as my theory gets more consistent.